src/django/pfb_analysis/models.py

```diff
@@ -19,15 +21,24 @@ class Neighborhood(models.Model):
     """Neighborhood boundary used for an AnalysisJob """
 
     def __repr__(self):
-        return "<Neighborhood: {}, ({})>".format(self.name, self.label)
+        return "<Neighborhood: {} ({})>".format(self.name, self.organization.name)
 
-    name = models.SlugField(max_length=256, primary_key=True)
+    uuid = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.SlugField(max_length=256, unique=True)
```

Comment:
Looking at this again before merging, I'm pretty sure that the SlugField is now superfluous. UUID for uniqueness at db level + unique_together on name (as human-readable CharField) + org is sufficient for our purposes. Thoughts?